projects/rhods/roles/rhods_delete_ods/tasks/main.yml

```diff
@@ -32,7 +32,7 @@
 - name: Get all the ODH/Kubeflow CRDs
   shell:
     set -o pipefail;
-    oc get crd -oname | grep -E 'opendatahub.io|kubeflow.org' | cut -d/ -f2
+    oc get crd -oname | (grep -E 'opendatahub.io|kubeflow.org' || true) | cut -d/ -f2
```

Comment:
The lint errors  are fixed in #145, this one

'the role 'check_deps' was not found in ansible.legacy:/tmp/roles:/home/runner/.cache/ansible-compat/fb6a86/roles:/home/runner/work/topsail/topsail/"./projects/repo/roles:/home/runner/work/topsail/topsail/projects/kubemark/roles:/home/runner/work/topsail/topsail/projects/rhods/roles:/home/runner/work/topsail/topsail/projects/local-ci/roles:/home/runner/work/topsail/topsail/projects/cluster/roles:/home/runner/work/topsail/topsail/projects/codeflare/roles:/home/runner/work/topsail/topsail/projects/llm_load_test/roles:/home/runner/work/topsail/topsail/projects/kepler/roles:/home/runner/work/topsail/topsail/projects/deprecated/ocm/roles:/home/runner/work/topsail/topsail/projects/deprecated/benchmarking/roles:/home/runner/work/topsail/topsail/projects/deprecated/entitlement/roles:/home/runner/work/topsail/topsail/projects/gpu-operator/roles:/home/runner/work/topsail/topsail/projects/notebooks/roles:/home/runner/work/topsail/topsail/projects/load-aware/roles:/home/runner/work/topsail/topsail/projects/kserve/roles:/home/runner/work/topsail/topsail/projects/lightspeed/roles:/home/runner/work/topsail/topsail/projects/pipelines/roles:/home/runner/work/topsail/topsail/":/home/runner/work/topsail/topsail/"./projects/repo/roles:/home/runner/work/topsail/topsail/projects/kubemark/roles:/home/runner/work/topsail/topsail/projects/rhods/roles:/home/runner/work/topsail/topsail/projects/local-ci/roles:/home/runner/work/topsail/topsail/projects/cluster/roles:/home/runner/work/topsail/topsail/projects/codeflare/roles:/home/runner/work/topsail/topsail/projects/llm_load_test/roles:/home/runner/work/topsail/topsail/projects/kepler/roles:/home/runner/work/topsail/topsail/projects/deprecated/ocm/roles:/home/runner/work/topsail/topsail/projects/deprecated/benchmarking/roles:/home/runner/work/topsail/topsail/projects/deprecated/entitlement/roles:/home/runner/work/topsail/topsail/projects/gpu-operator/roles:/home/runner/work/topsail/topsail/projects/notebooks/roles:/home/runner/work/topsail/topsail/projects/load-aware/roles:/home/runner/work/topsail/topsail/projects/kserve/roles:/home/runner/work/topsail/topsail/projects/lightspeed/roles:/home/runner/work/topsail/topsail/projects/pipelines/roles:/home/runner/work/topsail/topsail/":/home/runner/work/topsail/topsail/projects/cluster/roles:/tmp
'